### ISSUE TITLE
User Review

### ISSUE THREAD

**Issue by gkwan-ibm** [OpenLiberty/guide-grpc-intro#14]
User Review:

Allow a person with a fresh mind to read the guide and execute the steps described in the guide, test whether each step is clear enough to understand, and no logic errors.

Review and walkthrough by a non-SME or a new user with a critical and fresh pair of eyes:

- [x] Record the time it takes to go through the entire guide
- [x] Test each step in the guide is clear and easy to understand
- [x] Ensure no logic errors
- [x] Report anything that does not work
- [x] Report anything usability issues (confusing or hard to follow)
- [x] Does the guide achieve basic understanding/working knowledge of the topic in the guide?

**Comment by mbuamuh** [comment]
- [x] Record the time it takes to go through the entire guide 
It took me 45 minutes to go through the entire guide

**Comment by mbuamuh** [comment]
- [x]  Report anything that does not work 
The code works with no errors

**Comment by mbuamuh** [comment]
- [x] Test each step in the guide is clear and easy to understand
- [x] Ensure no logic errors
- [x] Report anything usability issues (confusing or hard to follow)

1.) Remarks about [https://guides-draft-openlibertyio.mqj6zf7jocq.us-south.codeengine.appdomain.cloud/guides/grpc-intro.html#what-youll-learn](url), paragraph 2:

- May be say something like `This guide consists of 3 maven project.` Then list the projects, saying what application or service the project consists of.
- I would have put what the `query` service implements under a new paragraph, in this section.
- The diagram in this section is confusing. What does it represent? A little bit of explanation would have been good. Also, maybe put the diagram under the section [https://guides-draft-openlibertyio.mqj6zf7jocq.us-south.codeengine.appdomain.cloud/guides/grpc-intro.html#creating-and-defining-the-grpc-service](url). A diagram says a thousand words, so maybe expand the diagram not only to include the query client methods, but also the system service methods that are called for each RPC call.

2.) Remarks about [https://guides-draft-openlibertyio.mqj6zf7jocq.us-south.codeengine.appdomain.cloud/guides/grpc-intro.html#implementing-the-unary-call](url):
- I would say `update` the server.xml file, rather than to `replace` it. 

3.) Remarks about [https://guides-draft-openlibertyio.mqj6zf7jocq.us-south.codeengine.appdomain.cloud/guides/grpc-intro.html#implementing-the-server-streaming-call](url):
- I would have said `Update the SystemService java class with the getPropertiesServer() method`. The wording `Replace the SystemService class.` can make one think they need to replace the file rather than update it.
- I would have said `Update the PropertiesResource java class with the getOSProperties() method`, rather than `Replace the PropertiesResource` class.
N.B: The above remarks are relevant for the sections [https://guides-draft-openlibertyio.mqj6zf7jocq.us-south.codeengine.appdomain.cloud/guides/grpc-intro.html#implementing-the-client-streaming-call](url), and [https://guides-draft-openlibertyio.mqj6zf7jocq.us-south.codeengine.appdomain.cloud/guides/grpc-intro.html#implementing-the-bidirectional-streaming-call](url) as well.

**Comment by mbuamuh** [comment]
- [x]  Does the guide achieve basic understanding/working knowledge of the topic in the guide?
I would have put the basic knowledge of gRPC as one of the prerequisites of this guide, or give more explanation about what gRPC is. I think without understanding what the gRPC framework is, its features and the use cases it solves, one might find it hard to follow the guide

**Comment by gkwan-ibm** [comment]
hi @shin19991207, would you fix the following?

- set the time to 30 mins
   - _(Note: 45 mins is too long)_ 

**What you’ll learn**
- " .... this guide consists of the `systemproto` model project, the `query` client service, and the `system` server service." -> "... this guide consists of three projects, the `systemproto` model project, the `query` client service project, and the `system` server service project. "
   - _(Note: not mention Maven project because we will support gradle in future)_ 
- It is ok to put "The query service implements ..." under a new paragraph
- I will revisit the diagram

**Implementing the unary call** (and other sections)
- no need to change "replace" to "update" because it is the guides practice to use the word "replace".

**Respond to**
> I would have put the basic knowledge of gRPC ...give more explanation...

- The guide has a section to talk about "What is gRPC?" at the beginning. The guide is not a reference/help document for gRPC. If users want to understand more about gRPC, they can visit the link at the beginning of the "What is gRPC?" section.

**Comment by shin19991207** [comment]
The above comments are addressed in https://github.com/OpenLiberty/draft-guide-grpc-intro/pull/15.
